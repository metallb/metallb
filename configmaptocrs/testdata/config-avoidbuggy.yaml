apiVersion: v1
kind: ConfigMap
metadata:
  namespace: metallb-system
  name: config
data:
  config: |
    address-pools:
    - name: my-ip-space
      protocol: bgp
      addresses:
      - 198.51.100.0/24
      - fc00:f853:0ccd:e799::/124
      avoid-buggy-ips: true
