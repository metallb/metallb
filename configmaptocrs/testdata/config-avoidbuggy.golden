# This was autogenerated by MetalLB's custom resource generator.
apiVersion: metallb.io/v1beta1
kind: IPAddressPool
metadata:
  creationTimestamp: null
  name: my-ip-space
  namespace: metallb-system
spec:
  addresses:
  - 198.51.100.1-198.51.100.254
  - fc00:f853:ccd:e799::1-fc00:f853:ccd:e799::e
status: {}
---
apiVersion: metallb.io/v1beta1
kind: BGPAdvertisement
metadata:
  creationTimestamp: null
  name: bgpadvertisement1
  namespace: metallb-system
spec:
  ipAddressPools:
  - my-ip-space
status: {}
---
